libs/fluid-elements/tab-group/src/lib/tab/tab.ts

```diff
@@ -148,13 +157,12 @@ export class FluidTab extends LitElement {
    * @attr
    * @type number
    */
-  @property({ type: Number, reflect: true })
+  @property({ type: Number, reflect: false })
   get tabindex(): number {
     return this._tabindex;
   }
   set tabindex(value: number) {
     this._tabindex = value;
-    // TODO: Figure out why using the old value does not work. The attribute vanishes in the dom/is not set
     this.requestUpdate('tabindex');
   }
```

Comment:
I'm curious, do we still need the getter and setter here? this seems like the default behavior.

Reply:
We don't need them actually.